Makefile

```diff
@@ -12,12 +12,13 @@ MKDIR_P ?= mkdir -p
 # Jenkins overwrites CPPFLAGS, so use CPPARGS instead.
 CPPARGS ?= $(INC_FLAGS) $(CPP_FLAGS) -MMD -MP -DVERSION="\"$(VERSION)\""
 
-m2cell: run_tests $(BIN_DIR)/m2cell
-
+# TODO: DM-  Make the executable in the 'serverMain' directory and copy it to bin.
+# Having main in the library broke the Catch2 unit tests.
+#m2cell: run_tests $(BIN_DIR)/m2cell
 # The main m2cell program.
-$(BIN_DIR)/m2cell: $(LIB_DIR)/libm2cellcpp.a
-	mkdir -p bin
-	$(CPP) $(OBJS) -o $@ $(LDFLAGS) $(CPP_LIBS)
+# $(BIN_DIR)/m2cell: $(LIB_DIR)/libm2cellcpp.a
+# 	mkdir -p bin
+# 	$(CPP) $(OBJS) -o $@ $(LDFLAGS) $(CPP_LIBS)
```

Comment:
Do you want to delete them since you could not use them in Catch2? I do not think to comment these code and leave them there is a good idea.